js/w3c/headers.js

```diff
@@ -155,6 +156,18 @@ define(
                     ret += ", <span class='ed_mailto'><a class='u-email email' " + rm + " href='mailto:" + p.mailto + "'>" + p.mailto + "</a></span>";
                 }
                 if (p.note) ret += " (" + p.note + ")";
+                if (p.extras && p.extras.length !== 0) {
+                    ret += ", ";
+                    for (var k = 0; k < p.extras.length; k++) {
+                        var extra = p.extras[k];
+                        if (extra.name) {       
+                            ret += extra.label ? "<span class='" + extra.label + "'>" : "<span>";
+                            ret += extra.href ? "<a" + rpe + " href='" + extra.href + "'>" + extra.name + "</a>" : extra.name;
```

Comment:
I'm a bit worried about extra.name as a XSS opportunity... need to check if we have a sanitize function.